src/styles/booksStyles.css

```diff
@@ -0,0 +1,211 @@
+.book-container {
+  margin: 0 6rem 0 6rem;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.single-book {
+  margin: 2rem 2rem 0 0;
+  padding: 0;
+  border: solid 1px rgba(129, 127, 127, 0.322);
+  display: flex;
+  justify-content: space-between;
+}
+
+.book-info {
+  width: 35%;
+  margin: 0 0 0 2rem;
+  padding: 0;
+}
+
+.book-category {
+  margin: 1.5rem 0 0.5rem 0;
+  font-family: "Montserrat", sans-serif;
```

Comment:
- [ ] You could consider using a reusable class for the `font-family` property as it reoccurs many times in your codebase  for CSS best practices

Reply:
Ok, thanks for the suggestion! I created the `.montserrat` class ✔️